understanding/consistent-help.html

```diff
@@ -53,322 +52,225 @@
                </p>
-               <p>Locating the help mechanism in a consistent location across pages makes it easier
-                  for users to find it.  For example, when a mechanism or link is located in the header
-                  of one Web page, it will be easier to find if it is in the header of other pages.
-                  The help mechanism, such as a contact phone number, may be provided directly on the
-                  page, or it may also be a direct link to a contact page. Regardless of which approach
-                  is used, the mechanism must be located in the same relative order on each page within
-                  the set of pages.
+               <p>ヘルプメカニズムをページ間で一貫した場所に配置すると、利用者が見つけやすくなる。例えば、あるメカニズム又はリンクが一つのウェブページのヘッダーにあるとき、それが他のページのヘッダーにある場合、見つけやすくなる。連絡先の電話番号などのヘルプメカニズムは、ページ上に直接提供されてもよく、又は連絡先ページへの直接リンクであってもよい。どのアプローチを使用するかに関係なく、そのメカニズムは、その一連のページ内の各ページに相対的に同じ順序で配置されなければならない。
                </p>
-               <p>When testing this Success Criterion, it is the help item which is relative to the
-                  rest of the content. When testing a page, other content that is present across the
-                  set of web pages and is before the help item should be before the help item on this
-                  page. Items which are after the help item on other pages should be after the help
-                  item on this page.
+               <p>この達成基準をテストする場合、残りのコンテンツと関連するヘルプ項目になる。ページをテストする場合、ウェブページ一式の全体に存在して、ヘルプ項目の前にある他のコンテンツは、このページのヘルプ項目の前に存在すべきである。他のページでヘルプ項目の後にある項目は、このページのヘルプ項目の後に存在すべきである。
                </p>
-               <p>If the help item is visually in a different location, but in the same serial order,
-                  that is not helpful from a user's point of view, but it would not fail this criterion.
+               <p>ヘルプ項目が視覚的には異なる場所にあるものの、同じシリアル順序で配置されている場合、利用者の観点からは役に立たないが、この基準に失敗することにはならない。
                </p>
-               <p>When having problems completing a task on a Web site (or part of a Web site, what
-                  we call a <a href="#dfn-set-of-web-pages">set of Web pages</a>), people with some types of disabilities may not be able to work through the issue
-                  without further help.  Issues could include difficulty:
-                  completing a form, or finding a document or page which provides information required
-                  to complete a task.
+               <p>ウェブサイト (又はウェブサイトの一部、いわゆる<a href="#dfn-set-of-web-pages">ウェブページ一式</a>) でタスクの完了に問題がある場合、障害のある人の中には、さらなるヘルプなしに問題を解決できない場合がある。問題には、フォームの完了、又は作業を完了するために必要な情報を提供する文書もしくはページを見つけることの難しさが含まれる可能性がある。
                </p>
-               <p>Without help, some users may abandon the task. They may also fail to correctly complete
-                  a task, or they may require assistance from people who do not necessarily keep private
-                  information secure.
+               <p>ヘルプがなければ、一部の利用者は作業をあきらめる可能性がある。また、作業を正しく完了するのに失敗する場合、又は個人情報を安全に管理しているとは限らない人からの支援を必要とする場合もある。
```

Comment:
ふたつめのセンテンスは、「(ヘルプがなければ)〜の可能性がある。」の列挙の続きのように思いました。こんな感じでしょうか？

```suggestion
               <p>ヘルプがなければ、一部の利用者は作業をあきらめる可能性がある。あるいは、作業を正しく完了するのに失敗する、又は個人情報を安全に管理しているとは限らない人からの支援を必要とする可能性もある。
```

Reply:
「あるいは」はorと紛らわしいので「同様に」としてみるのはどうでしょう。

```suggestion
               <p>ヘルプがなければ、一部の利用者は作業をあきらめる可能性がある。同様に、作業を正しく完了するのに失敗する、又は個人情報を安全に管理しているとは限らない人からの支援を必要とする可能性もある。
```

Reply:
よいと思います！